testdata/testcases/devices_tests.yml

```diff
@@ -0,0 +1,36 @@
+testSuite:
+  name: Devices tests
+  description: Test that devices should be mounted as volumes in the resulting generic worker task payload.
+  tests:
+    - name: Kvm and HostSharedMemory
+      description: >-
+        Tests that kvm and hostSharedMemory devices are mounted as volumes in the resulting generic worker task payload.
+      dockerWorkerTaskPayload:
+        command:
+          - echo "Hello world"
+        capabilities:
+          devices:
+            kvm: true
+            hostSharedMemory: true
```

Comment:
can we split this into two separate tests, with each one having one device capability set?